### PR TITLE
Instructions

### PR DESCRIPTION
**Pull request by tushartiwari7**
## Instruction modal before starting out the quiz.
ins

### Different quiz file for different type of quizzes.
<hr/>

landing

> RESULT PAGE
res

### CODE REVIEW

**Review by dev-enforced — CHANGES_REQUESTED** [review]
The code looks good to me.Had a small query mentioned at the bottom. Do deploy it on Netlify or Vercel so that if any error comes up during the live preview we can point to there too!!

File: style.css [hunk 1]
```diff
@@ -38,6 +43,18 @@ a {
   grid-area: main;
 }
 
+div.modal.modal {
```
Comment (dev-enforced) [inline style.css]: Here we are referring to the div that has a class of modal what is the second .modal for?

**Review by dev-enforced — CHANGES_REQUESTED** [review]

**Review by harshitadaswani — APPROVED** [review]
You can merge it

File: style.css (hunk 1 above)
Comment (tushartiwari7) [reply style.css]: To increase the specificity, in the CL i was using `div.modal` so i had to increase it even more hence ended up using same css selector twice.

**Comment by tushartiwari7** [comment]
Deployed Link: https://nature-quiz.vercel.app/

**Review by ashishnnnnn — APPROVED** [review]

**Review by dev-enforced — APPROVED** [review]
Yes the code looks good to merge!! The photos and deployed app looks great

**Pull request merged**